<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QG_DlgHatch</class>
 <widget class="QDialog" name="QG_DlgHatch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose Hatch Attributes</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QGroupBox" name="bgParameter">
       <property name="title">
        <string>Pattern</string>
       </property>
       <layout class="QGridLayout">
        <item row="2" column="1">
         <widget class="QLineEdit" name="leScale"/>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="leAngle"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lAngle">
          <property name="text">
           <string>Angle:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lScale">
          <property name="text">
           <string>Scale:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QG_PatternBox" name="cbPattern" native="true"/>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QCheckBox" name="cbSolid">
          <property name="text">
           <string>Solid Fill</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="bgPreview">
       <property name="title">
        <string>Preview</string>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QCheckBox" name="cbEnablePreview">
          <property name="text">
           <string>Enable Preview</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QG_GraphicView" name="gvPreview" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QG_PatternBox</class>
   <extends>QWidget</extends>
   <header>qg_patternbox.h</header>
  </customwidget>
  <customwidget>
   <class>QG_GraphicView</class>
   <extends>QWidget</extends>
   <header>qg_graphicview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbSolid</tabstop>
  <tabstop>leScale</tabstop>
  <tabstop>leAngle</tabstop>
  <tabstop>cbEnablePreview</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QG_DlgHatch</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QG_DlgHatch</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbSolid</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbPattern</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbSolid</sender>
   <signal>toggled(bool)</signal>
   <receiver>leScale</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbSolid</sender>
   <signal>toggled(bool)</signal>
   <receiver>lScale</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbSolid</sender>
   <signal>toggled(bool)</signal>
   <receiver>leAngle</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbSolid</sender>
   <signal>toggled(bool)</signal>
   <receiver>lAngle</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbPattern</sender>
   <signal>patternChanged()</signal>
   <receiver>QG_DlgHatch</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbSolid</sender>
   <signal>toggled(bool)</signal>
   <receiver>QG_DlgHatch</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leAngle</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QG_DlgHatch</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leScale</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QG_DlgHatch</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbEnablePreview</sender>
   <signal>toggled(bool)</signal>
   <receiver>QG_DlgHatch</receiver>
   <slot>updatePreview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
